cmake/setup_compiler_flags.cmake

```diff
@@ -78,9 +78,14 @@ SET(CACHED_DEAL_II_LINKER_FLAGS_SAVED "${DEAL_II_LINKER_FLAGS_SAVED}" CACHE INTE
 # Initialize all CMAKE_REQUIRED_* variables a this point:
 RESET_CMAKE_REQUIRED()
 
-CHECK_CXX_SOURCE_COMPILES(
-  "int main(){ return 0; }"
-  DEAL_II_HAVE_USABLE_CXX_FLAGS)
+# For some reason, the following fails for WIN64, disable it as a workaround.
+IF(${CMAKE_CL_64})
+	set(DEAL_II_HAVE_USABLE_CXX_FLAGS YES)
+ELSE()
+	CHECK_CXX_SOURCE_COMPILES(
+	  "int main(){ return 0; }"
+	  DEAL_II_HAVE_USABLE_CXX_FLAGS)
+ENDIF()
```

Comment:
As above. Why is it necessary to disable the sanity check on Win64 targets? What is the relevant error message?